apis/composition.yaml

```
@@ -211,6 +242,34 @@ spec:
                           cpu: 100m
                           memory: 256Mi
                     server:
+                      # https://docs.upbound.io/mcp/gitops/#add-the-up-cli-init-container-to-argo
+                      volumes:
+                        - name: up-plugin
+                          emptyDir: {}
+                        - name: up-home
+                          emptyDir: {}
+                      volumeMounts:
+                        - name: up-plugin
+                          mountPath: /usr/local/bin/up
+                          subPath: up
+                        - name: up-home
+                          mountPath: /home/argocd/.up
+                      initContainers:
+                        - name: up-plugin
+                          command:
+                            - sh
+                            - -c
+                          args:
+                            - wget -qO /plugin/up https://cli.upbound.io/stable/current/bin/${OS}_${ARCH}/up && chmod +x /plugin/up
```

```suggestion
                            - wget -qO /plugin/up https://cli.upbound.io/stable/current/bin/${OS}_${ARCH}/up && chmod +x /plugin/up
```
can we run something like `$(uname -m)` or `$(arch)` to dynamically determine the architecture from within container?